ansible/roles/fes-app-config/files/deployment-scripts/frontend_deployment.yml

```diff
@@ -1,4 +1,25 @@
 ---
+  - name: CA Certs
+    hosts: localhost
+    gather_facts: true
+    vars:
+      cert_name: "chca"
+    tasks:
+      - name: Download the cert artifact from S3
+        aws_s3:
+          bucket: "{{ s3_bucket_releases }}"
+          object: "chl-fes/{{ cert_name }}.pem"
+          dest: "/etc/pki/ca-trust/source/anchors/{{ cert_name }}.pem"
+          mode: get
+
+      - name: Update trust
+        shell: /usr/bin/update-ca-trust
```

Comment:
Can this be a command instead of a shell?

Reply:
changed to command